site/th.njk

```diff
@@ -0,0 +1,143 @@
+                    <p class="font-normal">
+                        เนื่องจากยังไม่มีภูมิคุ้มกันไวรัสตัวใหม่ ทุกคนจึงมีความเสี่ยงในการติดเชื้อ และแม้ว่าการเติบโตแบบทวีคูณอาจจะดูช้าในตอนแรก แต่มันจะสามารถเพิ่มขึ้นในระยะเวลาอันสั้นได้เช่นกัน
+                        ด้วยจำนวนผู้ติดเชื้อที่เพิ่มขึ้นทวีคูณในทุกๆวัน<sup><a class="source-link" href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/" target="_blank">[7]</a></sup>
+                        ระบบสาธารณสุขของเราอาจจะรองรับผู้ป่วยไม่ได้ ทำให้มีผู้เสียชีวิตที่มากขึ้น เพราะพวกเขาไม่ได้การรักษา
+                    </p>
+                </li>
+            </ol>
+
+            <p class="text-2xl">
+                ด้วยการร่วมขบวนการนี้ และลดความเป็นไปได้ที่จะเกิดการติดเชื้อขึ้นอีก คุณไม่ใช่แค่ปกป้องตัวเองเท่านั้น แต่คุณยังช่วยควบคุม และจำกัดการแพร่ระบาดให้กับผู้อื่นเช่นกัน โดยเฉพาะผู้ที่มีความเสี่ยงสูงที่จะเป็นโรคแทรกซ้อนหลังติดเชื้อ<br /><br />
+                <strong>ถ้าคุณเป็นห่วงคนอื่นในสถานการณ์ตอนนี้ กรุณาแชร์ลงโซเชียลมีเดีย หรือทางอีเมลล์ หรือแค่บอกเพื่อนและครอบครัวก็ได้ ขอแค่ #StayTheFuckHome</strong>
+            </p>
+
+            <div class="flex justify-center pb-12">
+                <a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-size="large" data-text="I will do my part in the fight against #COVID19. I urge you to do the same!" data-url="https://staythefuckhome.com" data-hashtags="StayTheFuckHome" data-show-count="false">Tweet</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+                <iframe class="ml-6" src="https://www.facebook.com/plugins/share_button.php?href=https%3A%2F%2Fstaythefuckhome.com&layout=button_count&size=large&appId=764658637392408&width=111&height=28" width="111" height="28" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true" allow="encrypted-media"></iframe>
+            </div>
+
+            <p class="text-sm text-gray-500 mb-10 mt-10">
+                For anyone who wants to contribute to the #StayTheFuckHome website and movement, we have open-sourced the website so everyone can contribute. The GitHub repository can be found here: <a href="https://github.com/flore2003/staythefuckhome" target="_blank">https://github.com/flore2003/staythefuckhome</a>
+                <br />
+                If you have any other input regarding the content of the website, please write us at <a href="mailto:contribute@staythefuckhome.com">contribute@staythefuckhome.com</a>.
+                Just keep in mind that we are currently receiving a lot of emails, so we might not get back to you directly.
+            </p>
+
+            <p class="text-sm text-gray-500 pt-12 max-w-4xl mx-auto">
+                <strong>Disclaimer:</strong> The content of this website does not constitute medical advice. The information provided above is meant as a suggestion for concerned individuals to follow a loose set of guidelines
```

Comment:
Disclaimer not translated.

Reply:
Thanks for the replies. I was pretty busy in previous day since moving my stuffs back from home so I tried translating the content first.